src/vm/ecalllist.h

```diff
@@ -1354,8 +1354,8 @@ FCFuncStart(gThreadFuncs)
 FCFuncEnd()
 
 FCFuncStart(gThreadPoolFuncs)
-#ifndef FEATURE_CORECLR
     FCFuncElement("PostQueuedCompletionStatus", ThreadPoolNative::CorPostQueuedCompletionStatus)
+#ifndef FEATURE_CORECLR
     FCFuncElement("GetAvailableThreadsNative", ThreadPoolNative::CorGetAvailableThreads)
```

Comment:
The great thing about model.xml is that you don't need to specify everything in the closure just the correct entry points

ThreadPool.GetAvailableThreads was not in my list. I can add it here but it is also part of https://github.com/dotnet/corefx/issues/11633 which is what @kouvel  is working on

Also the facade create by mscorlib.cs is no longer used by corefx and currently is only used by IL tests in coreclr [ I believe the mscorlib facade is either changing or going away some time from coreclr repo]